.vsts.pipelines/builds/matrix.yml

```diff
@@ -23,16 +23,27 @@ jobs:
 
 - template: ../jobs/ci-linux.yml
   parameters:
-    job: debian82
-    imageName: microsoft/dotnet-buildtools-prereqs:debian-8.2-4e6e894-20181023212416
+    job: centos8
+    imageName: microsoft/dotnet-buildtools-prereqs:centos-8-daa5116-20200325130212
```

Comment:
Should this be `mcr.microsoft.com/dotnet-buildtools/prereqs:centos-8-daa5116-20200325130212`? (MCR links for others too?)

Reply:
Right! 👍